.github/dependabot.yml

```diff
@@ -4,3 +4,11 @@ updates:
   directory: "/"
   schedule:
       interval: "weekly"
+
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: "weekly"
+  open-pull-requests-limit: 10
```

Comment:
limit to 10 can avoid bot sending too much PRs (which not very often I think but just limit here